dataclients/kubernetes/clusterclient.go

```diff
@@ -16,29 +16,31 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
+
+	"github.com/zalando/skipper/dataclients/kubernetes/definitions"
 	"github.com/zalando/skipper/secrets"
 )
 
 const (
 	ingressClassKey            = "kubernetes.io/ingress.class"
-	ingressesClusterURI        = "/apis/extensions/v1beta1/ingresses"
-	clusterZalandoResourcesURI = "/apis/zalando.org/v1"
-	routeGroupsName            = "routegroups"
+	IngressesClusterURI        = "/apis/extensions/v1beta1/ingresses"
+	ClusterZalandoResourcesURI = "/apis/zalando.org/v1"
```

Comment:
Can you rename this resource or the `IngressesClusterURI` to match a common scheme for both, please?
As we now open the const for library user we should have consistent names

Reply:
Done